### ISSUE TITLE
How many data points?

### ISSUE THREAD

**Issue by illogicalextend** [chartist-js/chartist#1077]
What is the maximum number of values that can be added in to a bar chart? I have 365 integers but the graph doesn't look correct. As if it imports up to a certain number then cuts off. Also, with this many points, the labels overlap like crazy so wonder if there is a way around this.

Or do I just need to aggregate data in to less values and play around myself to see how many I can fit in?

**Comment by stackasaur** [comment]
@illogicalextend Curious, if you have 365 values, I assume you are graphing something yearly? Meaning the labels would be temporal?

Would a line chart or time series not work for you?

**Comment by illogicalextend** [comment]
Yes, it is yearly data. The labels are dynamically populated, yes. 

I have just tried a line chart with the same data and it appears to correlate with the bar graph. So I guess it may be plotting all 365 points correctly, after all. 

But the labels are very screwed with this much data. Even if you try and add in 3 labels all together instead of 365, they are still printed all over each other and unreadable. Guess this is a limitation of chartist?

**Comment by Arantiryo** [comment]
Hello @illogicalextend 👋 

Chartist has recently been updated to version `1.0` with some new exciting changes like ESM and TypeScript support. The migration guide can be found right [here](https://github.com/chartist-js/chartist#whats-new-in-v1).

Please let me know if the question is still relevant to you so we can further look into it!
